flixel/addons/effects/FlxGlitchSprite.hx

```diff
@@ -69,17 +86,12 @@ class FlxGlitchSprite extends FlxSprite
 		}
 		else
 		{
-			_time += elapsed;
+			_time += FlxG.elapsed;
```

Comment:
elapsed only works if it comes through update, I was trying to avoid using update... any suggestions?